packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -328,6 +328,11 @@ void restoreAppStopper() {
 Future<bool> _stopApp(DriveCommand command) async {
   printTrace('Stopping application.');
   final ApplicationPackage package = await command.applicationPackages.getPackageForPlatform(await command.device.targetPlatform);
```

Comment:
I think I'm still a bit confused. Why wouldn't the package be there? Because there's no cmx file? For reference, the stopApp() for android doesn't have logic around this.

Reply:
This maybe because I ran without a cmx file. I will remove this for now, and re-add it if it needs to exist. I will restrict this PR to unifying cmx valdiation and moving the test-api dependency to test dependency.